model/certs_load_balancers.rb

```diff
@@ -0,0 +1,16 @@
+#  frozen_string_literal: true
+
+require_relative "../model"
+
+class CertsLoadBalancers < Sequel::Model
```

Comment:
Why did we create a model for this joint table?

Reply:
to override the `destroy` method

Reply:
IMHO, it's too much code to just override `destroy`. I feel like we can make it cleaner without `CertsLoadBalancers` pivot model. Let's get call for it today